io.github.Soundux.yml

```diff
@@ -0,0 +1,26 @@
+app-id: io.github.Soundux
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: soundux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
```

Comment:
you also need to --share=ipc if you use --socket=x11 (it allows x shared memory which is important for performance)

Reply:
Thank you for mentioning. I didn't know that.